asf_core_data/pipeline/data_joining/merge_proc_datasets.py

```diff
@@ -158,6 +176,9 @@ def add_mcs_installer_data(df, usecols=base_config.MCS_INSTALLER_FEAT_SELECTION)
         usecols = list(set(usecols + ["company_unique_id", "company_name"]))
 
     newest_hist_inst_batch = data_batches.get_latest_hist_installers()
+    # extracts "20230207" from "mcs_historical_installers_20230207.csv"
+    date = newest_hist_inst_batch.split("_")[-1].split(".csv")[0]
+    date = date[:4] + "-" + date[4:6] + "-" + date[6:]
```

Comment:
You could probably do this more elegantly with datetime. Something like this?
```suggestion
    date = str(datetime.strptime(date, "%Y%m%d").date())
```

Reply:
good point!